code/game/turfs/exterior/_exterior_icon.dm

```diff
@@ -0,0 +1,56 @@
+/turf/exterior/on_update_icon()
```

Comment:
Is this supposed to check height? It's called to update neighbors in height setting but doesn't seem to actually care about height differences.

Reply:
I think the height icon stuff is done in the trench PR itself?